src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import loginIcon from './login-icon.svg'
+import styles from './styles.css'
+
+function Button(props) {
```

Comment:
There's a reason why did you follow this pattern instead of creating a stateless arrow function? Like:
```
const Button = ({ onClick, text }) => (
```